client/app.jsx

```diff
@@ -31,9 +56,23 @@ export default class App extends React.Component {
 
   render() {
     return (
-      <Router history={this.props.history} onUpdate={this.onUpdate}>
-        {routes}
-      </Router>
+      // NOTE(2Pac): These are only the top-level routes. More specific routes are declared where
+      // they are used, as per react-router's new philosophy.
+      <Switch>
+        <Route path="/splash" component={Splash} />
+        <Route path="/faq" component={Faq} />
+        <Route path="/download" component={DownloadPage} />
+        <LoginRoute path="/forgot-password" component={ForgotPassword} />
+        <LoginRoute path="/forgot-user" component={ForgotUser} />
+        <LoginRoute path="/login" component={Login} />
+        <LoginRoute path="/reset-password" component={ResetPassword} />
+        <LoginRoute path="/signup" component={Signup} />
+        {devRoutes}
```

Comment:
Would be nice to just put these all under `/dev/` or something and have a Dev component we pull in that decides whether or not to expose them. Then you'd just need logic to decide whether to add that here, and not deal with building the actual list of dev routes in this file.

I can see you've *sort of* done that by putting them in different subject areas, but it'd probably be better to just have one Dev component at the root that we include (or don't) depending (or perhaps that just either returns a Switch or null depending? Would make the logic here simpler).